simtools/ray_tracing.py

```diff
@@ -350,6 +355,56 @@ def _isValidKeyToPlot(self, key):
         '''
         return key in ['d80_cm', 'd80_deg', 'eff_area', 'eff_flen']
 
+    def plotAndSave(self, key, **kwargs):
+        '''
+        Plot key vs off-axis angle and save the figure in pdf.
+
+        Parameters
+        ----------
+        key: str
+            d80_cm, d80_deg, eff_area or eff_flen
+        **kwargs:
+            kwargs for plt.plot
+
+        Raises
+        ------
+        KeyError
+            If key is not among the valid options.
+        '''
+        if not self._isValidKeyToPlot(key):
+            msg = 'Invalid key to plot'
+            self._logger.error(msg)
+            raise KeyError(msg)
+
+        ylabel = {
+            'd80_deg': r'$D_{80}$ [deg]',
+            'd80_cm': r'$D_{80}$ [cm]',
+            'd80_deg': r'$D_{80}$ [deg]',
+            'eff_area': 'Eff. Area [cm²]',
+            'eff_flen': 'Eff. Focal Length [cm]'
+        }
+
+        plotFileName = names.rayTracingPlotFileName(
+            key,
+            self._telescopeModel.telescopeType,
+            self._sourceDistance,
+            self._zenithAngle,
+            self.label
+        )
+        self._outputDirectory.joinpath('figures').mkdir(exist_ok=True)
+        plotFile = self._outputDirectory.joinpath('figures').joinpath(plotFileName)
+        self._logger.info('Plotting {} and saving it in {}'.format(key, plotFile))
+
+        plt.figure(figsize=(8, 6), tight_layout=True)
+        ax = plt.gca()
+        ax.set_xlabel('off-axis [deg]')
+        ax.set_ylabel(ylabel[key])
+
+        self.plot(key, **kwargs)
```

Comment:
I can't comment within "plot" cause it wasn't added in this commit, but why not use "visualize" in order to keep all plots with a consistent look?

Reply:
Yes, I agree. I just didn't learn yet to use the visualize, but it's a only matter of spending the time.
I will merge like this and create an issue to remind me to fix that later.